platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js

```diff
@@ -122,18 +122,18 @@ export default class WebSocket {
     this._onClose = null
   }
 
-  _addRef(){
-    CURR_REF_WEBSOCKET += 1
+  _addCount(){
+    CURR_AMOUNT_WEBSOCKET += 1
   }
 
-  _reduceRef(){
+  _reduceCount(){
```

Comment:
`add` and `reduce` are not antonyms. Maybe can use `increase` and `decrease`.

Reply:
OK